src/tests/api/StubTests.test.ts

```diff
@@ -1,8 +1,24 @@
-/* eslint-disable max-lines-per-function */
```

Comment:
Think we still need this - there are some linting warnings showing

Reply:
Added back in